code/modules/cargo/expressconsole.dm

```diff
@@ -108,32 +158,41 @@
 				rank = "Silicon"
 			var/reason = ""
 			var/list/empty_turfs
-			var/area/landingzone
 			var/datum/supply_order/SO = new(pack, name, rank, ckey, reason)
 			if(!(obj_flags & EMAGGED))
 				if(SO.pack.cost <= SSshuttle.points)
 					landingzone = locate(/area/quartermaster/storage) in GLOB.sortedAreas
-					for(var/turf/open/floor/T in landingzone.contents)
+					if (!landingzone)
+						WARNING("[src] couldnt find a Quartermaster/Storage area on the station, and as such has set the LZ to the area it resides in.")
+						landingzone = get_area(src)
+					for(var/turf/open/floor/T in landingzone.contents)//uses default landing zone
 						if(is_blocked_turf(T))
 							continue
 						LAZYADD(empty_turfs, T)
 						CHECK_TICK

```

Comment:
This `CHECK_TICK` means it might be possible for two invocations of this function to overlap; you need an additional check at the exact moment you remove credits (L182), otherwise spam-clicking can cause credits to go negative.